DSSE/environment/env.py

```diff
@@ -295,19 +287,30 @@ def create_observations(self):
 
         return observations
 
-    def calculate_simulation_time_step(self, drone_max_speed: float, cell_size: float, wind_resistance: float = 0.0) -> float:
+    def calculate_simulation_time_step(
```

Comment:
Redefining this function on person and dynamic_probability.

Reply:
Nice observation! Just extracted that function to a separate module.